stdlib/http/src/main/ballerina/src/http/common_client_config.bal

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shall we remove this content to http_commons.bal file as it contains all the commonly used types?

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/21275/commits/2bbf173f2e61c966f125812b80ddcdaf81b4ccff